source/modules/collection-api/transfer/actions/create.md.erb

```diff
@@ -1,27 +1,17 @@
 ### Transfer Creation
-
-```php
-<?php
-
-use Inspetor;
-
-$inspetor = $this->getConfiguredInspetor();
-
-$inspetor->trackItemTransferCreation($transfer);
-
-?>
-```
-```python 
-import inspetor
-
-inspetor = self.getConfiguredInspetor()
-
-inspetor.track_transfer_creation(transfer)
+```shell
+curl -XPOST https://collection-prod.inspcdn.net/transfer \
+    -H 'Authorization: Bearer <your API key>'   
+    -d '{
+        "id":             "43262",
```

Comment:
nit: is it intended to have some but not all of the values aligned in this block?